interface/forms/eye_mag/js/eye_base.php

```diff
@@ -369,7 +369,7 @@ function create_task(to_id,task,to_type) {
     var url = "../../forms/eye_mag/taskman.php";
     var formData = {
         'action'            : "make_task",
-        'from_id'           : '<?php echo $providerID; ?>',
+        'from_id'           : '<?php echo htmlspecialchars($providerID); ?>',
```

Comment:
This PR looks good and I'm going to bring this in now (expediting this since we are releasing a patch in a couple days and want to get this in there). I will change the above line to (since is within the js context):
```
'from_id'           : <?php echo js_escape($providerID); ?>,
```